### ISSUE TITLE
Fail to extract apk if `apk_output_path` parent folder is not created 

### ISSUE THREAD

**Issue by jam-cmesquita** [MartinGonzalez/fastlane-plugin-bundletool#1]
### Context
I'm running the plugin on my Android project and I want to save the apk file (`apk_output_path`) under `$HOME/bundletool_temp/universal_apk`, where `bundletool_temp` doesn't exist by default.

### What's wrong?
When I run the `bundletool` with the previous folder path, the function will throw the following error:
```
Bundletool could not extract universal apk from aab at /path/to/the/file.aab
```
If I'm not wrong, it is thrown at the `prepare_apk` function, probably at the `Dir.mkdir target_dir_name` or close to it. I know this, because I receive at least the logs printed at the line 94: `puts_important("Creating path #{target_path} since does not exist")`.

### Solution // Suggestion
So my assumption is that the plugin is only prepared to create the child folder (`universal_apk`) inside the  (supposedly created) parent folder (`bundletool_temp`).

I would suggest and it would be great to have in this plugin, a **way to create** not only the last child folder, but **the whole folder tree** (2 or more folder layers) provided as the `apk_output_path` if they don't exist at the moment.

**_PS:_** Maybe I'm wrong in my assumptions and I'm saying something wrong. If that's the case, please let me know how to fix it. Thank you!

**Comment by MartinGonzalez** [comment]
Hey! Thanks for filling an issue. 
I will test it to let you know if there is a problem.

Meanwhile are you sure that the `.aab` file exists in /path/to/the/file.aab ?

**Comment by jam-cmesquita** [comment]
Yes sure! I wrote the AAB path manually for testing.

In addition, after finding the reason why the plugin was throwing that error, the APK was generated successfully from the AAB.

**Comment by MartinGonzalez** [comment]
I've tests the following using the test in the fastfile.

The folder resources has only a `.aab` file so I configured in my action:

```ruby
lane :bundletool_test do
  bundletool(
    verbose: true,
    bundletool_version: '0.11.0',
    aab_path: 'resources/example.aab',
    apk_output_path: 'resources/output/example.apk'
  )
end
```

The folder output did not exists but it was created by the tool. Is not that the beahviour you expect?

EDIT:
I could reproduce the behaviour. I'll try to fix it now

**Comment by MartinGonzalez** [comment]
@jam-cmesquita Fixed in `1.0.1` :D
Thanks for reporting this! Please verify that this is what you wanted and if it is please close the issue :D

https://github.com/MartinGonzalez/fastlane-plugin-bundletool/blob/master/CHANGELOG.md

**Comment by jam-cmesquita** [comment]
It worked! And the error message is more complete now 😄 

Thanks a lot!
